packages/react-sdk/README.md

```diff
@@ -452,7 +452,8 @@ const useMessages: (
 };
 ```
 
-**Note:** It's important to memoize the `options` argument so that the hook doesn't fetch messages endlessly.
+> **Note**  
+> It's important to memoize the `options` argument so that the hook doesn't fetch messages endlessly.
```

Comment:
oops, we can actually remove this note. i made a change that no longer requires memoization.